reference/tests/snapshots/snapshot__run_reference_examples@introduction-0.prql.snap

```diff
@@ -30,10 +30,17 @@ WITH table_0 AS (
     20
 )
 SELECT
-  table_0.*,
+  title,
+  AVG(salary),
+  AVG(salary + payroll_tax),
+  SUM(salary),
+  SUM(salary + payroll_tax),
+  AVG(salary + payroll_tax + benefits_cost),
+  sum_gross_cost,
+  table_0.ct,
   countries.*,
   country_code,
-  true AS always_true,
```

Comment:
We don't parse boolean literals yet, so this was parsed as ident. Which now got resolved :D